demos/_includes/DemoLookup.php

```diff
@@ -29,7 +29,7 @@ protected function initQuickNewRecord()
 
         $buttonSeed = is_string($buttonSeed) ? ['content' => $buttonSeed] : $buttonSeed;
 
-        $defaultSeed = [\Atk4\Ui\Button::class, 'disabled' => ($this->disabled || $this->readonly)];
+        $defaultSeed = [\Atk4\Ui\Button::class, 'class.disabled' => ($this->disabled || $this->readonly)];
```

Comment:
⚠️ be careful, some components like `Atk4\Ui\Form\Control` has native `disabled` property